lib/services/location_service.dart

```diff
@@ -198,16 +198,16 @@ class LocationService {
 }
 
 class GPSData {
-  final String? latRef;
+  String? latRef;
```

Comment:
Why not keep this as final? I don't think modifying the value when toLocationObject is called is recommended approach. If needed, define additional variables inside `toLocationObject` or create a uil method `converToLocation(GPSData data)`